### PR TITLE
Adding metrics sessions IDs

### PR DESCRIPTION
**Pull request by lucaspasquier**
## GOAL
This PR is for adding a unique session Id for `core` and `container`.

## How to test
run unit test: `./gradlew clean test`

### CODE REVIEW

File: clappr/src/main/kotlin/io/clappr/player/utils/Environment.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package io.clappr.player.utils
+
+import java.util.*
+
+class Environment {
+    val sessionId = UUID.randomUUID().toString()
```
Comment (fcapinho) [inline clappr/src/main/kotlin/io/clappr/player/utils/Environment.kt]: `Core` is not directly connected to a single user or video session. Would this represent the `playerId` ?

File: clappr/src/main/kotlin/io/clappr/player/components/Container.kt [hunk 2]
```diff
@@ -44,6 +45,9 @@ class Container(options: Options) : UIObject() {
     override val viewClass: Class<*>
         get() = FrameLayout::class.java
 
+    val environment: Environment
```
Comment (fcapinho) [inline clappr/src/main/kotlin/io/clappr/player/components/Container.kt]: What is the use case for an always new `Environment` in `Container` ?

File: clappr/src/main/kotlin/io/clappr/player/utils/Environment.kt (hunk 1 above)
Comment (lucaspasquier) [reply clappr/src/main/kotlin/io/clappr/player/utils/Environment.kt]: done: 9c9a6aa0bdf9b5f663bdd4d142ca4a561b9db3fc

File: clappr/src/main/kotlin/io/clappr/player/components/Container.kt (hunk 2 above)
Comment (lucaspasquier) [reply clappr/src/main/kotlin/io/clappr/player/components/Container.kt]: done: cd59576152e8148577609d15989d8fce6fd13b83

File: clappr/src/main/kotlin/io/clappr/player/shared/SharedData.kt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package io.clappr.player.shared
+
+class SharedData {
+    val storeMap  = mutableMapOf<String, Any>()
```
Comment (fcapinho) [inline clappr/src/main/kotlin/io/clappr/player/shared/SharedData.kt]: Suggest declaring as a Map shortcut. Please refer to the `Options` class definition and its declaration inside the `Container` class.

Comment (mebarbosa) [reply clappr/src/main/kotlin/io/clappr/player/shared/SharedData.kt]: done: d79444e02f92dde4a2381a3ea52b330e1a8831ab

**Pull request merged**